pages/whats-new/all-news.md

```diff
@@ -0,0 +1,16 @@
+---
+title: All news and updates
+layout: post
+permalink: /about/whats-new/all/
+category: About
+type: posts
+---
+
+{% capture lead %}
+Here's the full listing of all USWDS product updates, articles, case studies and more.
+{% endcapture %}
+{% include lead.html text=lead %}
```

Comment:
It looks like this is the only page setting and including the lead in this way (the rest are done in `layout`.

**Suggestion (non-blocking):** Since this page isn't a post, it looks like we can use the `styleguide` layout and set the lead in the front matter like similar pages.

```suggestion
---
title: All news and updates
layout: styleguide
permalink: /about/whats-new/all/
category: About
lead: Here's the full listing of all USWDS product updates, articles, case studies and more.
type: posts
---
```